docs/docs/0.3.2/documentation/overview.md

```diff
@@ -61,6 +61,34 @@ When following the installation instructions, `kuma-injector` will be automatica
 <img src="/images/docs/0.2.0/diagram-08.jpg" alt="" style="width: 500px; padding-top: 20px; padding-bottom: 10px;"/>
 </center>
 
+ ### Matching Labels in `Pod` and `Service` 
```

Comment:
There is a space before `###`, won't it break anything?

Reply:
Good catch. I've removed it.